spec/services/search_service_spec.rb

```diff
@@ -75,9 +79,65 @@
     expect(NetworkMap).to receive(:search)
                             .with("@(title,description,index_data) foo",
                                   per_page: 50,
-                                  with: { is_deleted: false, is_private: false })
+                                  with: { is_deleted: false, is_private: false },
+                                  order: "is_featured DESC"
+                                 )
                             .once
 
     SearchService.new('foo').maps
   end
+
+  context 'with featured and non-featured lists' do
+    describe 'search', :sphinx do
+      before(:all) do # rubocop:disable RSpec/BeforeAfterAll
```

Comment:
I'd like to eventually refactor out all the `before(:all)` uses in our test suite. in the case, I think the ListEntity will stay in the test database, potentially causing issues. . I believe we can just use `before` here without any issues.